commit 978f989b20e08c9db3a05fe58e92b20088f991bc (refs/changes/31/2331/7)
Author: Tim Kordas <tim.kordas@joyent.com>
Date:   2018-01-22T13:21:12-08:00 (1 year, 9 months ago)
    
    MANTA-3364 'Loadbalancer has no backends' probe is missing tooling to actually run
    Reviewed by: David Pacheco <dap@joyent.com>
    Reviewed by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
    Approved by: Kelly McLaughlin <kelly.mclaughlin@joyent.com>

diff --git a/package.json b/package.json
index c95330f..da15204 100644
--- a/package.json
+++ b/package.json
@@ -11,6 +11,7 @@
         "dashdash": "1.14.1",
         "forkexec": "1.1.0",
         "jsprim": "1.4.0",
+        "haproxy-stat" : "0.1.0",
         "node-uuid": "1.4.1",
         "once": "1.4.0",
         "vasync": "2.0.0",
